app/src/main/java/com/yoshitoke/weatheringwithyou/mvp/model/DataClass/WeatherInfo.kt

```diff
@@ -0,0 +1,25 @@
+package com.yoshitoke.weatheringwithyou.mvp.model.DataClass
+
+import com.google.gson.annotations.SerializedName
+
+
+data class WeatherInfo(
+    @SerializedName("current")
+    val current: Current
+)
+
+data class Current(
+    @SerializedName("dt")
+    val dt: Int,
```

Comment:
what is dt? pls define a clearly variable

Reply:
it means datetime sir

Reply:
yes sir